### DISCUSSION TITLE
multiple instances of vrcx running at the same time

### DISCUSSION THREAD

**Discussion by hot-sauce9** [vrcx-team/VRCX#883] [Q&A]
so i used to be able to run more than 1 instance of vrcx since i had two accounts 
but seems like i cant do that anymore why is that 
and is it still possible to do it now?

**Comment by Natsumi-sama** [comment]
You need to run VRCX with a command line argument, you can do this by creating a shortcut and pasting it at the end of the path.
e.g. `--config="C:\VRCX"`
Each instance needs to use it's own database, so the path you set will create a database there, you could also copy over your old one but it wont update the original.
